# A message to recive pose of another agent
# and distance to it.

Header header
float32 range                               # range data [m]
geometry_msgs/PoseWithCovariance pose